derive_client/cli.py

```diff
@@ -14,6 +14,7 @@
 from rich.table import Table
 
 from derive_client.analyser import PortfolioAnalyser
+from derive_client import BaseClient
```

Comment:
Now there is 2 clients created in this cli tool no?

Reply:
it's just used as type annotation. Correct type annotation would be HttpClient or DeriveClient (the latter of which is just a wrapper for the HttpClient, and should arguably not exist)